torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6620,6 +6618,7 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            dtypesIfCPU=floating_types_and(torch.bfloat16),
            dtypesIfCUDA=floating_types_and(torch.half, torch.bfloat16),
            sample_inputs_func=sample_inputs_softmax_variant,
+           assert_jit_shape_analysis=True,
```

Comment:
It seems that you've added more shape functions than tests. Is it intentional?

Reply:
yea, see my other comment about softmax & layernorm being straightforward unary ops. happy to expand opinfo coverage for more complicate operators but would prefer not to do it for unary ops.